core/src/main/java/nu/yona/server/goals/service/ActivityCategoryService.java

```diff
@@ -53,18 +59,41 @@ public ActivityCategoryDTO getActivityCategory(UUID id)
 	@Transactional
 	public ActivityCategoryDTO addActivityCategory(ActivityCategoryDTO activityCategoryDTO)
 	{
-		logger.info("Adding activity category '{}'", activityCategoryDTO.getName(Translator.EN_US_LOCALE));
+		logger.info("Adding activity category '{}' with ID '{}'", activityCategoryDTO.getName(Translator.EN_US_LOCALE),
+				activityCategoryDTO.getID());
+		verifyNoDuplicateNames(Collections.emptySet(), activityCategoryDTO.getLocalizableNameByLocale());
```

Comment:
That's not a problem. Smoothwall already classifies sites under multiple categories, so if you do things that fall under multiple categories, you see activities under multiple categories. Summing up the activities might go beyond 100%